uniffi/src/interface/mod.rs

```diff
@@ -416,6 +416,13 @@ impl FFIFunction {
     pub fn return_type(&self) -> Option<&TypeReference> {
         self.return_type.as_ref()
     }
+    pub fn method_arguments(&self) -> Vec<&Argument> {
+        let args = self.arguments.get(1..self.arguments.len());
+        match args {
+            None => Vec::new(),
+            Some(args) => args.iter().collect(),
+        }
+    }
```

Comment:
```suggestion
    pub fn method_arguments(&self) -> &[Argument] {
        self.arguments.get(1..self.arguments.len()).unwrap_or_default()
    }
```

Super small nit, but maybe we can avoid the copy and `match` here?

Reply:
Oh that's soo much nicer! Thank you!